.internal-ci/helm/fog-ingest/templates/fog-ingest-statefulset.yaml

```diff
@@ -227,7 +229,7 @@ spec:
           - configMap:
               name: {{ include "fogIngest.fullname" . }}-supervisord-daemon
           - configMap:
-              name: fog-supervisord-mobilecoind
+              name: {{ include "fogIngest.fullname" . }}-supervisord-mobilecoind
```

Comment:
move the mobilecoind supervisord config from the config chart into the ingest. Its a bit of duplication, but helps us make the chart stand alone.